### ISSUE TITLE
Using torchreid data loading, I have lower result

### ISSUE THREAD

**Issue by SunTongtongtong** [cap-ntu/FedReID#7]
Hello,

The code is really clear and easy to read, thanks for sharing! 

However, due to the dataset process is not based on the original version(the img name in meta.json is different at least) which makes it a bit hard to follow, I use different data loading processes implemented based on torchreid. And the result I achieve is lower than reported in the paper(around 10 percent on average). I just wonder if the data loading can result in the degradation of the performance. 

Thanks for your reply,

**Comment by weimingwill** [comment]
Hi @SunTongtongtong, maybe you can try running it with only one dataset first to check whether the performance match with "Local Training" in the paper.

**Comment by Aotle** [comment]
> Hi, I am also having difficulties with data processing. I would be very grateful if you could share your data processing code.
Thanks for your reply,

**Comment by Huhongyi** [comment]
hello @SunTongtongtong ，I would like to ask you , How to deal with data set?
